scan_vin.go

```diff
@@ -31,15 +31,21 @@ func (p *scanVINCmd) SetFlags(f *flag.FlagSet) {
 	f.BoolVar(&p.send, "send", false, "send result over mqtt to the cloud")
 }
 
-func (p *scanVINCmd) Execute(_ context.Context, _ *flag.FlagSet, _ ...interface{}) subcommands.ExitStatus {
+func (p *scanVINCmd) Execute(_ context.Context, _ *flag.FlagSet, args ...interface{}) subcommands.ExitStatus {
```

Comment:
@JamesReate , does it look right?

Reply:
Hmm no this is part of google subcommands args. Maybe user passes in environment as dev or prod and depending it picks up right config file.

Reply:
Lookup github repo google subcommans

Reply:
Fixed, instead of passing the arguments, we can just read the file from locally, since earlier we wrote it on disk.
Btw, args carry over config as well